nixos/modules/services/misc/bazarr.nix

```diff
@@ -0,0 +1,80 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.bazarr;
+in
+{
+  options = {
+    services.bazarr = {
+      enable = mkEnableOption "bazarr";
+
+      dataDir = mkOption {
+        type = types.str;
+        default = "/var/lib/bazarr/data";
+        description = "The directory where bazarr stores its data files.";
+      };
+
+      openFirewall = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Open ports in the firewall for the bazarr web interface.";
+      };
+
+      listenPort = mkOption {
+        type = types.port;
+        default = 6767;
+        description = "Port on which the bazarr web interface should listen";
+      };
+
+      user = mkOption {
+        type = types.str;
+        default = "bazarr";
+        description = "User account under which bazarr runs.";
+      };
+
+      group = mkOption {
+        type = types.str;
+        default = "bazarr";
+        description = "Group under which bazarr runs.";
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    systemd.tmpfiles.rules = [
```

Comment:
Instead of using tmpfiles, you can use `systemd.services.bazarr.serviceConfig.StateDirectory`.

Reply:
Well, using `StateDirectory` means the whole `dataDir` option should be removed, because `StateDirectory` can only be a name and not a full path: the final state directory becomes `/var/lib/${StateDirectory}`.